experimental/bakeable/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "bakeable"
```

Comment:
nit: probably should be crabbake-derive and in crabbake/derive, see how yoke does it